tests/build_/test_build_request.py

```diff
@@ -1261,6 +1262,8 @@ def test_render_orchestrate_build(self, tmpdir, platforms, secret, disabled,
             assert kwargs.get('source_registry_uri') == worker_config.get_source_registry_uri()
             assert kwargs.get('sources_command') == worker_config.get_sources_command()
             assert kwargs.get('vendor') == worker_config.get_vendor()
+            assert (kwargs.get('equal_labels') or [] ==
```

Comment:
> The problem is we're not checking both options when we got or there:

We actually do.  If `equal_labels` is not a key in the provided dict, then we verify that the default value produced by `get_equal_labels` is used, which is `[]`.